### PR TITLE
Option for setting duration of log messages on slices and project pages

### LINKED ISSUE #1318: Allow retrieving more log messages
The home, project, and slice pages list 'recent' log actions at the bottom.
Currently, they all use the default, showing entries from the last 24 hours.

It would be nice to show more. Consider:
 - Allow an AJAX thing to "Show Older"
 - User setting of how  long back to look
 - Change the default to 72 hours (cover a weekend)

 _Imported from trac ticket [#1318](http://trac.gpolab.bbn.com/proto-ch/ticket/1318),  created by ahelsing on 02-19-2015 at 08:58, last modified: 02-27-2015 at 11:17_

### PR DESCRIPTION
**Pull request by charliemeyer**
The log tables on the project and slices pages now function the same as the log table on the homepage. (fixes #1318)

### CODE REVIEW

**Comment by ahelsing** [comment]
Code works as expected. But a couple clean-up items to do:

 - Remove the included changes to `tools-admin.php`
 - For consistency, make URL arg be `slice_id` not `sliceid` and make it `project_id` not `projectid`
 - Update `CHANGES` file to note this is changed on project and slice pages as well
 - Do the "No messages" thing on project/slice pages?

**Comment by charliemeyer** [comment]
Addressed comments above in the last commit there.

**Comment by ahelsing** [comment]
can you merge in the latest from develop again, and make sure your responsive CSS changes are in for the log message tables? When I try to merge I get conflicts in project.php, and I don't see the div for the responsive table on the slice page's log messages

File: portal/www/portal/slice.php [hunk 1]
```diff
@@ -798,34 +798,27 @@ function removeActiveBodies(a, b) {
...
+  function getLogs(hours){
+    url = "do-get-logs.php?hours="+hours+"&slice_id=" + <?php echo "\"" . $slice_id . "\""; ?>; 
+    $.get(url, function(data) {
+      $('#log_table').html(data);
+    });
   }
-?>
+</script>
 
-</table>
```
Comment (ahelsing) [inline portal/www/portal/slice.php]: Looks like you added the `<div>` around the table, but did not set the `class='tablecontainer'`. Shouldn't it have that as well?

**Pull request merged**